tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_cuda11_6_tensorrt8_4

```diff
@@ -0,0 +1,201 @@
+ARG BASEIMAGE=nvidia/cuda:11.6.1-cudnn8-devel-centos7
```

Comment:
Other docker files have the copyright info.

Reply:
will fix.